.evergreen/config.yml

```diff
@@ -759,6 +759,19 @@ axes:
            SSL: "nossl"
            AUTH: "noauth"
 
+  - id: csfle
+    display_name: "CSFLE"
+    values:
+      - id: csfle
+        display_name: CSFLE
+        variables:
+          CSFLE: "csfle"
+        batchtime: 10080  # 7 days
+      - id: nocsfle
```

Comment:
You don't need a 'nocsfle' value on this axis. If you are relying on the `$CSFLE` envvar to serve as a feature flag for installing pymongocrypt and running CSFLE tests, any situation where when that envvar is not set is equivalent to 'nocsfle'